src/core/Microsoft.PowerToys.Settings.UI.Lib/SettingsUtils.cs

```diff
@@ -114,8 +116,9 @@ public void SaveSettings(string jsonSettings, string powertoy = DefaultModuleNam
                     _ioProvider.WriteAllText(GetSettingsPath(powertoy, fileName), jsonSettings);
                 }
             }
-            catch
+            catch (Exception e)
             {
+                Logger.LogError($"Exception encountered while saving {powertoy} settings.", e);
```

Comment:
Wonder if I should still rethrow Argument/ArgumentNull exceptions so they can be caught by programmers at development time?

Reply:
We can throw them in debug mode but in release mode we should handle it since saving settings to a json file for one powertoy shouldn't affect others.

Reply:
We can throw them in debug mode but in release mode we should handle it since saving settings to a json file for one powertoy shouldn't affect others.